frontstage/views/surveys/add_survey_submit.py

```diff
@@ -49,4 +59,22 @@ def add_survey_submit(session):
     logger.info('Successfully retrieved data for confirm add organisation/survey page',
                 case_id=case_id, party_id=party_id)
     return redirect(url_for('surveys_bp.get_survey_list', _anchor=(business_party_id, added_survey_id), _external=True, business_party_id=business_party_id,
-                            survey_id=added_survey_id, tag='todo'))
+                            survey_id=added_survey_id, tag='todo', already_enrolled=already_enrolled))
+
+
+def is_business_enrolled(associations, survey_id, party_id):
```

Comment:
This function name should probably be `is_respondent_and_business_enrolled` as it's really checking 2 things.